math-scala/src/main/scala/org/apache/mahout/math/algorithms/regression/LinearRegressorModel.scala

```diff
@@ -54,46 +58,72 @@ trait LinearRegressorFitter[K] extends RegressorFitter[K] {
     addIntercept = hyperparameters.asInstanceOf[Map[Symbol, Boolean]].getOrElse('addIntercept, true)
   }
 
+
   def calculateStandardError[M[K] <: LinearRegressorModel[K]](X: DrmLike[K],
                              drmTarget: DrmLike[K],
                              drmXtXinv: Matrix,
                              model: M[K]): M[K] = {
     import org.apache.mahout.math.function.Functions.SQRT
     import org.apache.mahout.math.scalabindings.MahoutCollections._
-    var modelOut = model
+
     val yhat = X %*% model.beta
     val residuals = drmTarget - yhat
-    val ete = (residuals.t %*% residuals).collect // 1x1
+
+    // Setting modelOut.rss
+    // Changed name from ete, to rssModel.  This is residual sum of squares for model of yhat vs y
+    var modelOut = calculateResidualSumOfSquares(model,residuals)
+
     val n = drmTarget.nrow
     val k = safeToNonNegInt(X.ncol)
     val invDegFreedomKindOf = 1.0 / (n - k)
-    val varCovarMatrix = invDegFreedomKindOf * ete(0,0) * drmXtXinv
+    val varCovarMatrix = invDegFreedomKindOf * modelOut.rss * drmXtXinv
     val se = varCovarMatrix.viewDiagonal.assign(SQRT)
     val tScore = model.beta / se
-    val tDist = new org.apache.commons.math3.distribution.TDistribution(n-k)
+    val tDist = new TDistribution(n-k)
+
     val pval = dvec(tScore.toArray.map(t => 2 * (1.0 - tDist.cumulativeProbability(Math.abs(t))) ))
+
     // ^^ TODO bug in this calculation- fix and add test
     //degreesFreedom = k
-
-
     modelOut.se = se
     modelOut.tScore = tScore
     modelOut.pval = pval
-    modelOut.degreesFreedom = X.ncol
-    modelOut.summary = generateSummaryString(modelOut)
+    // for degrees of freedom, dont count the intercept term that was added
+    modelOut.degreesFreedom = X.ncol - 1
+
+    modelOut.trainingExamples = n.toInt
```

Comment:
This is dangerous- could be to big of a number to fit in an Int. `safeToNonNegInt(residuals.nrow)`